app/scripts/controllers/settings.js

```diff
@@ -87,6 +87,21 @@ angular.module('openshiftConsole')
           Logger.log("quotas", $scope.quotas);
         });
 
+        DataService.list("appliedclusterresourcequotas", context, function(quotas) {
+          $scope.clusterQuotas = quotas.by("metadata.name");
+          $scope.emptyMessageQuotas = "There are no resource quotas set on this project.";
```

Comment:
Hm, then it will switch to the message for both when one loads even if the other is still pending?